src/writer/follow.ts

```diff
@@ -1,6 +1,24 @@
+import { FOLLOWS_LIMIT_PER_USER } from '../helpers/limits';
 import db from '../helpers/mysql';
 
-export async function verify(): Promise<any> {
+export const getFollowsCount = async follower => {
```

Comment:
Based on this function name it should return a number and not an array of object